src/Diagnostics.RuntimeHost/appsettings.json

```diff
@@ -54,19 +54,23 @@
     "HeartBeatTimeOutInSeconds": 10,
     "HeartBeatDelayInSeconds": 20,
     "AADAuthority": "https://login.windows.net/microsoft.com",
-    "AADKustoResource": "https://wawskustotest.kusto.windows.net"
+    "AADKustoResource": "https://wawskustotest.kusto.windows.net",
+    "Enabled": true
```

Comment:
We have to update these new settings to our key vault right.. Will it cause any breaking change ?

Reply:
No, whatever configuration is not in KeyVault is picked up from the AppSettings